### DISCUSSION TITLE
Essentials.VersionTracking.IsFirstLaunchEver on app migrated from XF to MAUI

### DISCUSSION THREAD

**Discussion by munkii** [dotnet/maui#22458] [Q&A] [answered]
We are migrating XF to MAUI and already use Essentials.VersionTracking. I can see from the MAUI docs [how to configure VersionTracking](https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/appmodel/version-tracking?view=net-maui-8.0).

Those same docs say that the preferences are stored with a filename of  `[YOUR-APP-PACKAGE-ID].microsoft.maui.essentials.versiontracking`. Seeing as the name has "maui" in it i'm guessing the check IsFirstLaunchEver will be true when the user upgrades unknowingly from the XF version to the MAUI version or have you added some migration code in there?

Is that correct?

**Comment by phillippschmedt** [comment, marked as answer]
https://learn.microsoft.com/en-us/dotnet/maui/migration/version-tracking?view=net-maui-8.0#consume-legacy-version-tracking-data
